fetch.bs

```diff
@@ -4597,22 +4598,21 @@ steps:
       <p class=note>The `<code>GET</code>` <a for=/>method</a> restriction serves no useful purpose
       other than being interoperable.
 
-     <li><p>Let <var>response</var> be a new <a for=/>response</a> whose
-     <a for=response>status message</a> is `<code>OK</code>`.
+     <li><p>Let <var>bodyWithType</var> be the result of <a for=BodyInit>safely extracting</a>
+     <var>blobURLEntry</var>'s <a for="blob URL entry">object</a>.
```

Comment:
Hmm. "extract"'s Blob processing is much vaguer than https://w3c.github.io/FileAPI/#blob-get-stream . What is the plan for when to use one versus the other? Should File API just use "safely extract" and get rid of "extract stream"? Basically, who should own the blob -> stream logic, File API or Fetch?

Reply:
Oh, I guess https://github.com/whatwg/fetch/pull/1506 fixes this, nevermind.